pom.xml

```diff
@@ -33,4 +33,17 @@
         </dependency>
     </dependencies>
 
+  <build>
```

Comment:
Can you please move build section above dependencies and align indentation with the rest of the document?